issue4/about.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <nav class="nav-bar-container">
+            <div class="logo-container div--bg-grey">
+                <p>Logo</p>
+            </div>
+            <div class="nav-btn-container">
+                <a href="/index.html">
```

Comment:
One observation I had, the `<a />` tag has `cursor: pointer`, and the `<button />` tag has `cursor: default`. As a result, only the area between the buttons shows the cursor. I would play around with the CSS a bit to get the `cursor: pointer` working on the button itself.

Reply:
Resolved by adding cursor: pointer to nav button class.